packages/xrpc-server/src/auth.ts

```diff
@@ -116,18 +124,13 @@ export const verifyJwt = async (
 
   const msgBytes = ui8.fromString(parts.slice(0, 2).join('.'), 'utf8')
   const sigBytes = ui8.fromString(sig, 'base64url')
-  const verifySignatureWithKey = async (key: string) => {
-    return crypto.verifySignature(key, msgBytes, sigBytes, {
-      jwtAlg: header.alg,
-      allowMalleableSig: true,
-    })
-  }
```

Comment:
Unless there is a strong reason not to do so, I would advocate that we change this function here to be implemented this using `node:crypto` instead of `@atproto/crypto`.